lib/config.yml

```diff
@@ -19,21 +19,48 @@ Rails:
 Bundler/OrderedGems:
   Enabled: false
 
+Layout/AlignParameters:
+  EnforcedStyle: with_fixed_indentation
+
 Layout/CaseIndentation:
   EnforcedStyle: end
 
 Layout/ElseAlignment:
   Enabled: false
 
+Layout/EmptyLinesAroundBeginBody:
+  Enabled: false
+
 Layout/EmptyLinesAroundBlockBody:
   Enabled: false
 
+Layout/EmptyLinesAroundClassBody:
+  Enabled: false
+
 Layout/EmptyLinesAroundExceptionHandlingKeywords:
   Enabled: false
 
+Layout/FirstArrayElementLineBreak:
+  Enabled: true
+
+Layout/FirstHashElementLineBreak:
+  Enabled: true
+
+Layout/FirstMethodArgumentLineBreak:
```

Comment:
I think the idea is that you don't get code like this:

```ruby
a_long_method_name(foo,
  bar,
  baz
)
```

Might as well just put `foo` one line down.